src/TextArea/index.js

```diff
@@ -337,7 +340,7 @@ Textarea.propTypes = {
   value: PropTypes.string,
   label: PropTypes.string,
   disabled: PropTypes.bool,
-  error: PropTypes.string,
+  error: PropTypes.oneOfType([PropTypes.bool, PropTypes.string]),
```

Comment:
Looks good.

I did notice that we failed to include a `PropType` definition for `validators` (namely, `validators: PropTypes.object`) and `validationErrors` (`validationErrors: PropTypes.object`), and now is probably as good a time as any to add that?